index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
O que você acha de alterar a lang?

```suggestion
<html lang="pt-br">
```